packages/dev/loaders/src/BVH/bvhLoadingOptions.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Options for loading BVH files
+ */
+export type BVHLoadingOptions = {
+    /**
+     * Defines the name of the animation to load.
+     */
+    animationName: string;
+
+    /**
+     * Defines the loop behavior of the animation to load.
+     */
+    loopBehavior: number;
```

Comment:
Is this actually used? I see it flowing down through a few layers, but I don't see it actually being read.

Reply:
Yes, this is used very internally: https://github.com/BabylonJS/Babylon.js/pull/16540/files#diff-2d5b0fc5f4668038627e0439f176f627d49394590f4aad816cbc30805162e94eR39

Reply:
It's assigned to the context (I think this is what you are pointing to with the link above), but then nothing reads it from the context, correct? Or am I still missing something?

Reply:
You are 100% right! I renamed it to loopMode and now I am assigning it properly.